src/material-table.js

```diff
@@ -55,8 +55,8 @@ export default class MaterialTable extends React.Component {
   setDataManagerFields(props, isInit) {
     let defaultSortColumnIndex = -1;
     let defaultSortDirection = '';
-    if (props) {
-      defaultSortColumnIndex = props.columns.findIndex(a => a.defaultSort);
+    if (props.options.sorting !== false) {
```

Comment:
You may check for props is not undefined. // (props && props.options.sorting !== false)

Reply:
Though i've tested passing nothing on the props and it prompted nothing, yes i'd apply now safe checking. thanks a lot for the review!